.travis.yml

```diff
@@ -13,7 +13,7 @@ before_install:
 install:
   - wget -q https://raw.githubusercontent.com/qiime2/environment-files/master/latest/staging/qiime2-latest-py36-linux-conda.yml
   - conda env create -q -n test-env --file qiime2-latest-py36-linux-conda.yml
-  - source activate test-env
+  - conda activate test-env
```

Comment:
You'll need to run `conda init` to get the shell configured, first.

Reply:
`conda init` will use bash by default, and Catalina uses `zsh` IIRC. When/if we add a darwin workflow to this sucker, is that something we should be dealing with? If so, I can open an issue.

Reply:
Note: Based on the official docs (and a failed attempt to use `conda init` here), we probably won't have access to `conda init` and its defaults. if we want to handle multiple shells, we may have to explicitly source the shell-appropriate scripts. See [SO](https://stackoverflow.com/questions/55342122/conda-activate-on-travis-ci).